wavefront/server/modules/plugins_module/plugins_module/controllers/cloud_storage_controller.py

```
@@ -0,0 +1,47 @@
+
+
+@cloud_storage_router.get('/v1/storage/signed-url')
+@inject
+async def get_resource_presigned_url(
+    resource_url: str = Query(..., description='The cloud storage URL of the resource'),
+    expires_in: int = Query(
+        300, description='Expiry time in seconds for the presigned URL'
+    ),
+    response_formatter: ResponseFormatter = Depends(
+        Provide[CommonContainer.response_formatter]
+    ),
```

Ruff (0.15.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)